src/test/java/uk/gov/hmcts/reform/notificationservice/ApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.notificationservice;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ApplicationTest {
+    @Test
+    void dummyTest() {
```

Comment:
Why can't we have health endpoint test in integration ?

Reply:
It is a unit test - just to pass checks

Reply:
You can write an integration tests to verify health.